Dockerfile

```diff
@@ -52,24 +52,6 @@ RUN install_packages \
 # have to be empty. So we delete them.
 RUN rm -rf /var/log/nginx && rm -rf /run
 
-# Gem inventory:
```

Comment:
I've moved these below the python deps because I think I'll manipulate them more frequently.